### PR TITLE
Create messages.hi.js

### PR DESCRIPTION
**Pull request by Shubhdeep02**
Closes https://github.com/chesslablab/website/issues/313

### CODE REVIEW

**Comment by programarivm** [comment]
Thank you @Shubhdeep02 for contributing to this repo!

It seems as if the `Draw by fivefold repetition` text has already been translated at https://github.com/chesslablab/website/issues/300

Let me point out the following:

- Before sending a PR, make sure to first `git pull` to fetch the latest changes from the remote repository and merge them into your local branch.

See:

- [Should I git pull first before making any changes in Git?](https://www.quora.com/Should-I-git-pull-first-before-making-any-changes-in-Git)

File: assets/js/translations/messages.hi.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const messagesHi = {
+  // Abstract
+  "Draw by the fifty-move rule": "पचास चालों के नियम द्वारा ड्रॉ",
+  "Draw by stalemate": "स्टेलमेट द्वारा ड्रॉ",
+  "Draw by fivefold repetition": "पाँच गुना पुनरावृत्ति द्वारा ड्रॉ",
```
Comment (programarivm) [inline assets/js/translations/messages.hi.js:5]: It seems as if the `Draw by fivefold repetition` text has already been translated at https://github.com/chesslablab/website/issues/300. Before sending a PR, make sure to first `git pull` to fetch the latest changes from the remote repository and merge them into your local branch.

Comment (Shubhdeep02) [reply assets/js/translations/messages.hi.js:5]: Ah!!!! extremely sorry for that part. i will revert the commit and update the changes for the PR

Comment (Shubhdeep02) [reply assets/js/translations/messages.hi.js:5]: @programarivm is it fine now?

Comment (programarivm) [reply assets/js/translations/messages.hi.js:5]: Excellent, thanks!

**Pull request merged**